src/components/extensive/Modal/ModalImageDetails.tsx

```diff
@@ -102,6 +102,7 @@ const ModalImageDetails: FC<ModalImageDetailProps> = ({
         pathParams: { uuid: data.uuid },
         body: {
           name: formData.name,
+          //@ts-ignore
```

Comment:
Why is this needed? If it is necessary, please use `// @ts-expect-error` with an explanation. Example: https://github.com/wri/wri-terramatch-website/blob/bc340bdcbbeb93c80978fdb69a5551c2ca210453/src/admin/apiProvider/dataProviders/projectDataProvider.ts#L21